examples/configs-demo/usersecrets_slightfile.toml

```diff
@@ -10,7 +10,7 @@ value = "g88GVRuP1o2aqE6L1Mg"
 
 [[secret_settings]]
 name = "THIS_IS_ANOTHER_TEST_CONFIG_1061248"
-value = "jIybsBA7MIDM7B3VIjA"
+value = "_MSeeJP5ivT67vPUkwM"
```

Comment:
Yeah~ I've talked to Joe about this and I think we've decided not to take any actions on it yet. 

That's because this is **technically**  a feature — i.e., we all have different keys that were randomly generated by the usersecrets implementor of the configs capability, so the value will change as we each make PRs.